tests/integration/ha_tests/test_upgrade_from_stable.py

```diff
@@ -26,11 +26,16 @@
 @pytest.mark.abort_on_fail
 async def test_deploy_stable(ops_test: OpsTest) -> None:
     """Simple test to ensure that the PostgreSQL and application charms get deployed."""
-    await ops_test.model.deploy(
+    await ops_test.juju(
```

Comment:
This will be rolled back as soon as a revision is published to the `16/beta` channel.